templates/publisher/listing.html

```diff
@@ -259,9 +255,9 @@
                     placeholder="mailto:example@example.com" />
                 </div>
                 {% if field_errors and field_errors['contact'] %}
-                  <p class="p-form-validation__message">
-                    <strong>Error:</strong> {{ field_errors['contact'] }}
-                  </p>
+                <p class="p-form-validation__message">
```

Comment:
Doesn't matter much (we don't really have code practices around that I think), but I would say that for code readability we should indent templates from point of view of template and template tags (so anything inside `if` should be indented) - not from point of view of resulting HTML.

Reply:
done